README.md

```diff
@@ -85,11 +83,6 @@ docker pull ghcr.io/dipdup-io/stone-packaging/stone-prover:latest
 
 ### Creating and Verifying a Test Proof Using Docker
 
-Clone the repository:
```

Comment:
Let's download https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#download-binaries-for-x86_64 instead of cloning, we still need to clone the repo to create a verify a test proof